editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5741,17 +5741,20 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 				sprite->set_texture(texture);
 				sprite->set_modulate(Color(1, 1, 1, 0.7f));
 				preview_node->add_child(sprite);
-				label->show();
-				label_desc->show();
+				dropping_tooltip_label->show();
+				dropping_tooltip_label_desc->show();
+				dropping_tooltip_label->set_text(vformat(TTR("Adding %s..."), default_texture_node_type));
 				desc += "\n" + TTR("Hold Alt + Shift when dropping to add as a different node type.");
-				label_desc->set_text(desc);
+				dropping_tooltip_label_desc->set_text(desc);
 			} else {
 				if (scene.is_valid()) {
 					Node *instance = scene->instantiate();
 					if (instance) {
 						preview_node->add_child(instance);
-						label_desc->show();
-						label_desc->set_text(desc);
+						dropping_tooltip_label->show();
+						dropping_tooltip_label_desc->show();
+						dropping_tooltip_label->set_text(vformat(TTR("Adding Scene...")));
```

Comment:
```suggestion
						dropping_tooltip_label->set_text(TTR("Adding Scene..."));
```

Reply:
I was'nt sure why there was a `vformat` in some of the labels; what is it for?

Reply:
It's to format, replaces things like `%s`